mmv1/third_party/terraform/services/compute/resource_compute_snapshot_settings_test.go

```diff
@@ -0,0 +1,74 @@
+package compute_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccComputeSnapshotSettings_snapshotSettings_basic(t *testing.T) {
```

Comment:
nit: usually we call this `_update` to indicate it's an update test.

```suggestion
func TestAccComputeSnapshotSettings_snapshotSettings_update(t *testing.T) {
```